.vib/kafka/goss/goss.yaml

```diff
@@ -2,32 +2,30 @@
 # SPDX-License-Identifier: APACHE-2.0
 
 file:
-  {{ .Vars.persistence.mountPath }}:
+  {{ .Vars.controller.persistence.mountPath }}:
     exists: true
     filetype: directory
     mode: "2775"
     owner: root
-  {{ .Vars.logPersistence.mountPath }}:
+  {{ .Vars.controller.logPersistence.mountPath }}:
     exists: true
     filetype: directory
     mode: "2777"
     owner: root
-  /opt/bitnami/kafka/config/kraft/server.properties:
+  /opt/bitnami/kafka/config/server.properties:
     exists: true
     filetype: file
-    mode: "0664"
-    owner: root
-    group: root
+    mode: "0644"
     contains:
```

Comment:
I'm missing connectivity and capability testing. See the example of Milvus goss.yaml. If there are metrics it would be nice to test them too.

Reply:
Isn't `create-kafka-topic` connectivity and capability testing?

Reply:
I see these ports:

```
service:
  ports:
    client: 9092
    controller: 9093
    interbroker: 9094
    external: 9095
```

If these are changed, does it work?

Reply:
Fixed an issue with the tests when changing ports and added new parameters to modify the service ports.